src_c/draw.c

```diff
@@ -695,7 +695,7 @@ circle(PyObject *self, PyObject *args, PyObject *kwargs)
     }
 
     if (width < 0) {
-        return RAISE(PyExc_ValueError, "negative width");
+        return pgRect_New4(posx - radius, posy - radius, 0, 0);
```

Comment:
I think in this case the rect's x, y values should just be the circle's center (posx, posy). You will also need to update the test case for this.

From the circle's documentation.
> if nothing is drawn the bounding rect's position will be the ``center`` parameter value (float values will be truncated) and its width and height will be 0

Reply:
Sure, makes sense. I'll update this.

Reply:
Added a commit to address this change.